### PR TITLE
l10n: Spelling unification

### PR DESCRIPTION
**Pull request by Valdnet**
Spelling unification in Transifex.

Signed-off-by: Valdnet <47037905+Valdnet@users.noreply.github.com>

### CODE REVIEW

File: lib/Activity/Provider.php [hunk 1]
```diff
@@ -78,7 +78,7 @@ public function parse($language, IEvent $event, IEvent $previousEvent = null) {
 				$event->setSubject($l->t('You removed an WebAuthn hardware token'));
 				break;
 			case 'webauthn_disabled_by_admin':
-				$event->setSubject($l->t('WebAuthn disabled by an admin'));
+				$event->setSubject($l->t('WebAuthn disabled by an administrator'));
```
Comment (rakekniven) [inline lib/Activity/Provider.php:81]: 'WebAuthn disabled by an administrator' -> 'WebAuthn disabled by the administration'

**Review by rakekniven — CHANGES_REQUESTED** [review]

File: lib/Activity/Provider.php [hunk 2]
```diff
@@ -72,13 +72,13 @@ public function parse($language, IEvent $event, IEvent $previousEvent = null) {
 		$event->setIcon($this->urlGenerator->getAbsoluteURL($this->urlGenerator->imagePath('core', 'actions/password.svg')));
 		switch ($event->getSubject()) {
 			case 'webauthn_device_added':
-				$event->setSubject($l->t('You added an WebAuthn hardware token'));
+				$event->setSubject($l->t('You added WebAuthn hardware token'));
```
Comment (rakekniven) [inline lib/Activity/Provider.php:75]: You can have multiple hardware tokens.
So If you added a new one "you added an hardware token".
@Valdnet Does that make sense?

Comment (Valdnet) [reply lib/Activity/Provider.php:75]: Hmm… I looked at the text strings in Transifex and the messages are written differently. Some are with the indefinite article "an" and some are without the indefinite article. Following your line of thought, we can leave this text string unchanged 😉.

**Review by rakekniven — CHANGES_REQUESTED** [review]

File: lib/Activity/Provider.php [hunk 3]
```diff
@@ -72,7 +72,7 @@ public function parse($language, IEvent $event, IEvent $previousEvent = null) {
 		$event->setIcon($this->urlGenerator->getAbsoluteURL($this->urlGenerator->imagePath('core', 'actions/password.svg')));
 		switch ($event->getSubject()) {
 			case 'webauthn_device_added':
-				$event->setSubject($l->t('You added WebAuthn hardware token'));
+				$event->setSubject($l->t('You added an hardware token'));
```
[the remark this answers is not in the document]
Comment (Valdnet) [reply lib/Activity/Provider.php]: I have already corrected 😉.

File: lib/Activity/Provider.php [hunk 4]
```diff
@@ -75,10 +75,10 @@ public function parse($language, IEvent $event, IEvent $previousEvent = null) {
 				$event->setSubject($l->t('You added an WebAuthn hardware token'));
 				break;
 			case 'webauthn_device_removed':
-				$event->setSubject($l->t('You removed an WebAuthn hardware token'));
```
Comment (rakekniven) [inline lib/Activity/Provider.php:78]: Same here as written in my previous post. 
Please revert and stay with "an".

**Review by rakekniven — CHANGES_REQUESTED** [review]

File: lib/Activity/Provider.php (hunk 4 above)
Comment (Valdnet) [reply lib/Activity/Provider.php:78]: Done 😉.

**Review by rakekniven — APPROVED** [review]

**Pull request merged**